### PR TITLE
Trip page itinerary

### PR DESCRIPTION
**Pull request by aionate0812**

### CODE REVIEW

File: src/components/Itinerary/Itinerary.js [hunk 1]
```diff
@@ -30,18 +28,18 @@ const  Itinerary = (props) => {
     console.log(categories)
```
Comment (pamelaabreu) [inline src/components/Itinerary/Itinerary.js]: Are these consoles necessary?

**Review by pamelaabreu — CHANGES_REQUESTED** [review]

File: src/assets/country_codes.json [hunk 2]
```diff
@@ -0,0 +1,252 @@
+{
```
Comment (JoshuaKGoldberg) [inline src/assets/country_codes.json:1]: It'd probably be easier to get these from an npm package. https://www.google.com/search?q=npm+package+country+codes&oq=npm+package+country+codes&aqs=chrome..69i57.2032j1j7&sourceid=chrome&ie=UTF-8 gives a few good ones; [`i18n-iso-countries`](https://www.npmjs.com/package/i18n-iso-countries) seems pretty popular.

File: src/containers/Trip/Trip.js [hunk 3]
```diff
@@ -70,17 +70,22 @@ class Trip extends Component {
...
-                    <h1>{ city }, { country }</h1>
-                    <div className='col-1'></div>
-                    <div className='' style={{}}>
-                        <p style={{ margin:'0' }}><span>{ departure_date.format('L') }</span> - <span>{ return_date.format('L') }</span></p>
-                        <p style={{ textAlign:'center', margin:'0' }}>{ moment().endOf(departure_date).to(return_date) }</p>
+                <div className='row justify-content-between'>
+                    <div className='col-lg-4' style={{}}>
+                        <div>
+                            <h5>Trip Details</h5>
+                            <h1 style={{fontSize:'3.5rem'}}>{ city }, { country }</h1>
```
Comment (JoshuaKGoldberg) [inline src/containers/Trip/Trip.js]: > `style={{fontSize:'3.5rem'}}`

Is there a reason you're putting these styles inline? It's generally better to use classes in your `.css` files for reusability and clarity.

File: src/containers/Trip/Trip.js [hunk 4]
```diff
@@ -70,17 +70,22 @@ class Trip extends Component {
...
+                        </div>
+                        <div className='' style={{}}>
+                            <p style={{}}>{ moment().endOf(departure_date).to(return_date) }</p>
```
Comment (JoshuaKGoldberg) [inline src/containers/Trip/Trip.js]: > `style={{}}`
> `className=''`

Nit: you can get rid of all of these; they do nothing.

File: src/containers/Trip/Trip.js (hunk 3 above)
Comment (JoshuaKGoldberg) [inline src/containers/Trip/Trip.js]: An `h1` shouldn't be used as a visual child within an `h5`. The numeric value of these headings is their semantic importance. "Trip Details" is more important than "City, Country" and so it should have a smaller number.

Judging by the `WeatherCard` component, it looks like the `h1` should be an `h6`?

If your goal is to make "City, Country" heading larger than "Trip Details" (which is totally reasonable), it'd be better to use CSS.

**Review by JoshuaKGoldberg — DISMISSED** [review]
Mostly looking great! 👍 Just some requested cleanups around semantic HTML.

**Comment by JoshuaKGoldberg** [comment]
Reposting here for visibility:

> Btw, if you disagree with one of my reviews, you can always dismiss it or re-request review in the UI. I'm using it to show how I would review this at work, but you definitely shouldn't be blocked on me being wrong about something. 😄
> 
> image

File: src/containers/Trip/Trip.js (hunk 3 above)
Comment (aionate0812) [reply src/containers/Trip/Trip.js]: Question,
Would I have to change the `h1` to a `p` tag?

**Review by pamelaabreu — APPROVED** [review]

**Review by pamelaabreu — DISMISSED** [review]
Is there anyway to not have inline styling and instead have a class w the styling you need.

File: src/containers/Trip/Trip.js (hunk 3 above)
Comment (JoshuaKGoldberg) [reply src/containers/Trip/Trip.js]: @aionate0812 it's still semantically a header - you can switch it to an `h6` and that should be fine.

Comment (aionate0812) [reply src/containers/Trip/Trip.js]: I see, so have it as an `h6` but just make it bigger using css?

Comment (JoshuaKGoldberg) [reply src/containers/Trip/Trip.js]: Exactly!

**Comment by JoshuaKGoldberg** [comment]
@pamelaabreu here's an example of a `.js` file in this repo that imports a `.css` file: https://github.com/pamelaabreu/bundle_frontend/blob/1751ec78e380ebf34ee85b31402a2e4231f76cbf/src/containers/SuggestionsModal/SuggestionsModal.js#L2. That's what I mean.

**Review by FiveEightyEight — APPROVED** [review]
LGTM

**Pull request merged**